File: common_java_lib/common/src/com/whaley/biutil/BiEventLog.java
package com.whaley.biutil;

/**
 * Created by wanglei on 2016/7/28.
 */
public class BiEventLog extends BiLog {
    public BiEventLog(String caseId)
    {
	super("event");
	PutHeader("eventId", caseId);
    }
}

File: common_native_lib/bieventlog.cpp
/*
 * Yuchuan Sun
 * Version 1.0
 * Date 2016-8-5
 * BiLog common module that is used by native code
*/

#define LOG_TAG "BiEventLog"
#include <utils/Log.h>

#include "bieventlog.h"

BiEventLog::BiEventLog(const char * caseId):
    BiLog("event")
{
    ALOGI("constructor");
    PutHeader("eventId",caseId);
}

BiEventLog::~BiEventLog()
{
    ALOGI("destructor");
}

